src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -8,6 +8,7 @@ import cromwell.engine.backend.jes.Run.{Failed, Running, Success, _}
 import cromwell.engine.db.{JesStatus, JesId, JesCallBackendInfo, DataAccess}
 import cromwell.util.google.GoogleScopes
 import org.slf4j.LoggerFactory
+import DataAccess._
```

Comment:
Okay, I was perhaps improperly conflating my objections here.  I don't like the name `slickDataAccess` when the type is `DataAccess`, as this does not reference anything special about the `SlickDataAccess` type.  I defer to your experience with overly magical implicit pain, let's do what we can to avoid that.  :smile:  `globalDataAccess` sounds fine to me.